### DISCUSSION TITLE
possible copypaste error w/ syncperiod

### DISCUSSION THREAD

**Discussion by secretagentmoof** [erichelgeson/BlueSCSI-v1#19] [General]
in BlueSCSI.cpp, around line 1206, there's
```c++
        // Transfer period factor(50 x 4 = Limited to 200ns)
        syncperiod = m_msb[i + 3];
        if (syncperiod > 50) {
          syncoffset = 50;
        }
```
should that be syncperiod = 50?

**Comment by marius137** [comment]
Because you are adding random amounts to the variable it will never Hit exactly 50. Its probably best to Change it to `if( syncperiod >= 50)` so the If includes 50

**Reply by erichelgeson** [reply]
If the if included 50 and syncperiod was 50 - it would just re-set the variable to 50 - i think `>` is ok unless I am missing something.

**Reply by marius137** [reply]
True, my bad. I thought the issue was about the if clause itself, instead of the content of the if. I really should stop reading git issues while still half asleep.

**Reply by erichelgeson** [reply]
No worries! - more eyes the better :)

**Comment by erichelgeson** [comment]
Good catch! - it looks like it - I'll give it a try later and see if it breaks/fixes anything :)

**Reply by erichelgeson** [reply]
Just a quick update as i tested it over lunch - I think it's probably correct. I'll test it with a few other machines and dig a bit more before committing it.
